spec/operations/books_lists/destroy_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'faker'
+
+describe ::BooksLists::Destroy do
+  subject(:creates_list) { ::BooksLists::Create.call(book:, list:) }
+  subject(:destroy_list) { ::BooksLists::Destroy.call(book:, list:) }
```

Comment:
By default, the subject is the object being tested. So, in your case you should have one subject and let name it accordingly (for example service, operation, result):
`subject(:service) { described_class.call(book:, list:) }`